code/variables.tf

```diff
@@ -334,3 +334,30 @@ variable "adb2c_audience" {
   type        = string
   description = "recipients that the JWT is intended for"
 }
+
+## Email
+variable "email_host" {
+  type        = string
+  description = "email server hostname"
+  default     = "fast.smtpok.com"
+}
+
+variable "email_port" {
+  type        = number
+  description = "email server port"
+  default     = 80
```

Comment:
why not 443?

Reply:
I took the port from a test account of mailup smtp <img width="615" alt="mailup-smtp" src="https://user-images.githubusercontent.com/1236551/118125292-1248ff80-b3f7-11eb-82be-48aec4dd59a6.png">

Port 25 was not working in azure app service.